### PR TITLE
Version bump: net-libs/nodejs-0.12.4

### PR DESCRIPTION
**Pull request by jbergstroem**
Introduce some minor functionality, otherwise straightforward.

### CODE REVIEW

**Comment by jbergstroem** [comment]
hm, dosym doesn't need `||die` on newer eabi's, right? also, missing newline on last line :X

**Comment by jbergstroem** [comment]
Come to think of it, cctest is probably broken, but we already restrict tests.

**Comment by mgorny** [comment]
Yes, built-in `do*` functions don't need ||die.

**Comment by mgorny** [comment]
Fix those two you mentioned, and I'll merge it.

File: net-libs/nodejs/nodejs-0.12.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+src_install() {
+	local LIBDIR="${ED}/usr/$(get_libdir)"
+	emake install DESTDIR="${ED}" PREFIX=/usr
+	use npm && dodoc -r "${LIBDIR}"/node_modules/npm/html
+	rm -rf "${LIBDIR}"/node_modules/npm/{doc,html} || die
+	find "${LIBDIR}"/node_modules -type f -name "LICENSE*" -or -name "LICENCE*" -delete
+
+	# set up a symlink structure that npm expects..
+	mkdir -p "${ED}/usr/include/node/deps/{v8,uv}" || die
```
Comment (mgorny) [inline net-libs/nodejs/nodejs-0.12.3.ebuild]: Likewise :).

**Comment by mgorny** [comment]
```
+*nodejs-0.12.4-r1 (26 May 2015)
+
+  26 May 2015; Michał Górny <mgorny@gentoo.org> +nodejs-0.12.4-r1.ebuild,
+  -nodejs-0.12.4.ebuild:
+  Remove BrE-spelled 'LICENCE' files. Introduce a node-gyp-friendly include
+  structure. Made from https://github.com/gentoo/gentoo-portage-rsync-
+  mirror/pull/124 by jbergstroem.
```

**Pull request closed without merging**